packages/core/ui/styling/style-scope.ts

```diff
@@ -214,17 +212,20 @@ class CSSSource {
 		if (this._source) {
 			switch (parser) {
 				case 'css-tree':
+					const cssTreeParse = require('../../css/css-tree-parser').cssTreeParse;
 					this._ast = cssTreeParse(this._source, this._file);
-
 					return;
 				case 'nativescript':
+					const CSS3Parser = require('../../css/parser').CSS3Parser;
+					const CSSNativeScript = require('../../css/parser').CSSNativeScript;
```

Comment:
I think this should also work with tree-shaking?
```suggestion
					const { CSS3Parser, CSSNativeScript } = require('../../css/parser');
```